tests/chains/ethereum/scripts/cmd/helper/chain.go

```diff
@@ -0,0 +1,266 @@
+package helper
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"math/big"
+	"os"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+	gethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/client"
+	ibccommitment "github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ibccommitmenttesthelper"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ibchandler"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ics20bank"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ics20transferbank"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/simpletoken"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/wallet"
+)
+
+type PathConfig struct {
+	Src      PathInfo `json:"src"`
+	Dst      PathInfo `json:"dst"`
+	Mnemonic string   `json:"mnemonic"`
+}
+
+type PathInfo struct {
+	ChainID      string `json:"chain-id"`
+	ClientID     string `json:"client-id"`
+	ConnectionID string `json:"connection-id"`
+	ChannelID    string `json:"channel-id"`
+	PortID       string `json:"port-id"`
+	Order        string `json:"order"`
+	Version      string `json:"version"`
+}
```

Comment:
You can `PathEnd` defined in `yui-relayer/core/pathEnd.go` instead of `PathInfo`.